content/booklet.md

```diff
@@ -0,0 +1,178 @@
+
+### Hardware
+Each team will have exactly one
+workstation.
+The only operating system available to you will be **Ubuntu 24.04** with **Xfce**.
+
+### Software
+
+The jury will run your submissions on the same system with the same compilers.
+During the contest, you will be able to choose between the languages **C, C++, Python, Java and Kotlin**.
+Using different languages for different problems is allowed.
+For a complete list of available IDEs, editors, language versions and compiler commands, see [2024.nwerc.eu/systems](/systems).
+
+### During the contest
+An introductory presentation on the systems will be given on Saturday; please refer to the schedule for the exact time.
+At this presentation, do not hesitate to ask any questions.
+Furthermore, if you stumble upon any problems with the system during the test session or contest, let us know and we will be more than happy to assist you.
+
+![Da Vinci](/da_vinci_ad.png)
+
+## Jury and Rules
+We follow the World Finals rules, so a team is not allowed to bring any reference materials such as books, program listings, or notes (exceptfor the TRDs)!
+You may not bring any electronic device or machinereadable media.
+You may bring pens or pencils, but not paper.
+Winter coats, bags etc. should not be brought to the contest area.
+
+Each contestant may bring a copy of a Team Reference Document (TRD).
+This may contain up to 25 pages of reference materials, singlesided, letter or A4 size.
+The pages must be numbered in the upper right-hand corner and contain the university name in the upper lefthand corner.
+Text and illustrations must be readable by a person with correctable eyesight without magnification from a distance 0.5 meter.
+It may include handwritten comments and corrections on the front of the pages only.
+The document should be in a notebook or folder with the name of your institution on front.
+
+Each team member may bring one printed, unannotated natural language dictionary.
+You may bring mascots such as stuffed animals or party hats provided they do not violate the above constraints.
+If there is any other item you need, please contact the Head of Jury.
+
+Teams are allowed to bring one keyboard.
+Note that keyboards with pre-programmed macro keys are not allowed.
+Wireless keyboards are also not allowed.
+One mouse is allowed too, but is subject to the same macro rule as with the keyboard.
+For more jury advice, see: [2024.nwerc.eu/jury-advice](/jury-advice).
+
+### Team Composition
+A student may compete when eligible according to the official ICPC regional rules summarised in the 2024 ICPC Regionals Eligibility Diagram.
+A team consists of 3 eligible students.
+The educational institutions must be from one of the region's member countries.
+By special request, teams from other countries may be allowed to participate in this region, as stated in the ICPC Regional Rules.
+All teams who want to participate must be registered.
+
```

Comment:
Add piece about Conduct? e.g.
```suggestion

### Conduct during contest
The ICPC is an extra-curricular, competitive programming sport of the universities of the world. 
Good sportsmanship is a requirement. 
At NWERC, teams and coaches should not wear or bring anything that might distract from spotlighting any other team.
At the contest, opening ceremony and award ceremony require official NWERC Shirts, do not bring anything with you other than smiles, goodwill, and teammates. 
Badges should be worn visibly at all times.
The regional contest director is empowered to add additional restrictions and to correct inappropriate behavior.

```

Reply:
> The regional contest director is empowered to add additional restrictions and to correct inappropriate behavior.

Since this is our own booklet, should we either replace this sentence by the actual additional restrictions that we posed, or remove it entirely if we have no additional restrictions?

This paragraph is also on the rules page already, as a quote. In the quote, it does make sense to have that sentence, since there it's clear that the text comes from ICPC.

Reply:
So do we add it or not?

Reply:
I would add it, unless @mpsijm objects

Reply:
I agree with Verwoerd's reasoning, so it's fine by me to add :slightly_smiling_face: 

(I'll check the rest of the updates to the booklet later)